locales/it-IT/messages.po

```diff
@@ -391,3 +391,5 @@ msgstr "Il tuo sistema non è ottimale, si prega di contattare il proprio ammini
 msgid "Your system requires a compatibility update, please contact your system administrator."
 msgstr "Il tuo sistema richiede un aggiornamento, si prega di contattare il tuo amministratore di sistema"
 
+msgid "This browser is supported."
+msgstr "Questo browser è supportato"
```

Comment:
In Portuguese, "browser" is called "navegador", I've been looking at examples of "navigatore" for Italian. Which one do we keep?